presto-docs/src/main/sphinx/sql/grant.rst

```diff
@@ -0,0 +1,51 @@
+=====
+GRANT
+=====
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    GRANT ( privilege [, ...] | ( ALL PRIVILEGES ) )
+    ON [ TABLE ] table_name TO ( grantee | PUBLIC )
+    [ WITH GRANT OPTION ]
+
+Usage of the term ``grantee`` denotes both users and roles.
+
+Description
+-----------
+
+Grants the specified privileges to the specified grantee.
+
+Specifying ``ALL PRIVILEGES`` grants ``DELETE, INSERT`` and ``SELECT`` privileges.
+
+Specifying ``PUBLIC`` grants privileges to all grantees.
+
+The optional ``WITH GRANT OPTION`` clause allows the grantee to grant these same privileges to others.
+
+For ``Grant`` statement to succeed, the user executing it should possess the specified privileges as well as the ``GRANT OPTION`` for those privileges.
+
+Limitation
+----------
+
+``GRANT`` is currently supported only for hive connector.
```

Comment:
Here is the reason I added the limitation here:
``GRANT/REVOKE`` statements are currently supported _only_ for the hive connector. If we don't mention this on the syntax page, I think that would be misleading. It will give an impression that this statement is supported by a significant number of connectors, which is not the case. 

If we decide otherwise to add this to connector documentation, this should be mentioned as a limitation for each and every connector except Hive connector. And the "Limitations" section at the bottom of the connector's pages can be easily missed by readers. What do you think?